hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/has/getevmaddressalias/EvmAddressAliasCall.java

```diff
@@ -61,7 +62,7 @@ public EvmAddressAliasCall(@NonNull final HasCallAttempt attempt, final Address
         final var accountNum = numberOfLongZero(explicitAddress);
         final var account = enhancement.nativeOperations().getAccount(accountNum);
         // If the account is null or does not have an account id then return bail
-        if (account == null || !account.hasAccountId()) {
+        if (account == null) {
```

Comment:
`// If the account is null or does not have an account id then return bail` -> `// if the account is null then tell user there's no such account` or something like that